test/overall_test.py

```diff
@@ -208,6 +208,22 @@
 >>> del(g)
 >>> del g_.data1
 >>> del(g_)
+>>> g1 = GenerativeImage()
+>>> function1 = eval("lambda x,y:" + g1.function1_str)
+>>> function2 = eval("lambda x,y:" + g1.function2_str)
+>>> g2 = GenerativeImage(function1=function1, function2=function2)
+>>> g1.generate(seed=22)
+>>> g2.generate(seed=22)
+>>> is_same_data(g1.data1, g2.data1)
+True
+>>> is_same_data(g1.data2, g2.data2)
+True
+>>> len(g1.data1)>0
+True
+>>> len(g1.data2)>0
```

Comment:
```
>>> len(g1.data1) > 0
True
>>> len(g1.data2) > 0
```

Reply:
fixed in https://github.com/sepandhaghighi/samila/pull/126/commits/c9b14e1ac8f8533be3917348d7105e3cedd9a19f